Readme.md

```diff
@@ -21,7 +21,7 @@ resp, err:= client.Execute(request)
 
 // or
 
-m := make(map[string]interface{})
+responseMap := make(map[string]interface{})
 err:= client.ExecuteInto(request, &m)
```

Comment:
`err:= client.ExecuteInto(request, &responseMap)`, This also needs to be refactored.